### DISCUSSION TITLE
PufferPanel starting on ipv6

### DISCUSSION THREAD

**Discussion by mikedoesgithub** [pufferpanel/pufferpanel#1269] [General]
i have everything setup as far as users, so i go to run it and after its done i get "[INFO] 2023/08/07 11:24:59 Listening for HTTP requests on [::]:5692" i have reinstalled pufferpanel and a new os 4 times.

**Comment by LordRalex** [comment]
What OS are you using?

It seems like you changed the port, so what is your config?

**Comment by ayubrawr** [comment]
I have this exact problem too, any fix?
